.changeset/warm-drinks-prove.md

```diff
@@ -0,0 +1,6 @@
+---
+'@swisspost/design-system-styles': minor
+'@swisspost/design-system-documentation': minor
+---
+
+Created a hint documentation page and used `post-assist` token for global style of form hints.
```

Comment:
Since the component styles haven't changed externally, there's no need for a changelog in the styles package:
```suggestion
---
'@swisspost/design-system-documentation': minor
---

Created a documentation page for the form hints.
```